maxim/logger/langchain/utils.py

```
@@ -274,6 +281,21 @@ def parse_langchain_chat_generation(generation: ChatGeneration):
     return choices
 
 
+def parse_langchain_generation_chunk(generation: GenerationChunk):
```

Ruff (0.11.9)

Missing return type annotation for public function `parse_langchain_generation_chunk`

(ANN201)